packages/jsapi-utils/src/TableUtils.test.ts

```diff
@@ -615,8 +1038,10 @@ describe('quick filter tests', () => {
     it('handles invalid cases', () => {
       const column = makeFilterColumn();
 
+      // @ts-expect-error text is generally a string
       expect(() => TableUtils.makeQuickDateFilter(column, null, '')).toThrow();
       expect(() =>
+        // @ts-expect-error text is generally a string
         TableUtils.makeQuickDateFilter(column, undefined, '')
       ).toThrow();
```

Comment:
Don't need cases where we pass in `null` or `undefined` as TS shouldn't allow that.